tools/site_and_regional/neon_gcs_upload

```diff
@@ -0,0 +1,171 @@
+#! /usr/bin/env python3
+"""
+Script to rename and upload NEON site finidat files for use in transient startup cases
+
+"""
+
+import os, sys
```

Comment:
I think minimally we should do a pylint check of this new script as well as run it through black.

Reply:
We could also move the guts of it to the python directory, so that python pylint and black checking will be done automatically. This would also make it easier (and possible) to do unit and system testing. I don't think that's necessary for this script, but it's good to make it easier to do. It's good to have our important scripts done that way. It does seem like this is an important script for NEON, but it is only important for NEON, so it's not general purpose.

Reply:
I'll start this and make sure it can be done quickly.

Reply:
I started work on this, and found that this script requires a specific python environment. I wasn't able to easily get a conda environment setup using our py_env_create script. @wwieder has separate instructions to do that. This is also a script that will only be used by a limited set of people, so it's OK if it doesn't follow the same standards as other scripts.

Reply:
OK, we decided not to do this. So resolving this point.